install_gridpack_deps.sh

```diff
@@ -71,62 +84,87 @@ then
   # Download, build, and install GA
   cd ${GP_EXT_DEPS}
 
-  echo "Downloading GA-5.8"
+  if [ "$install_shared" == "true" ]; then
+      gashopts="--enable-shared=yes --enable-static=no"
+  else
+      gashopts="--enable-shared=no --enable-static=yes"
+  fi
+
+  gaversion="5.8.2"
+  echo "Downloading GA-$gaversion"
 
-  wget https://github.com/GlobalArrays/ga/releases/download/v5.8/ga-5.8.tar.gz
+  wget "https://github.com/GlobalArrays/ga/releases/download/v${gaversion}/ga-${gaversion}.tar.gz"
 
-  tar -xf ga-5.8.tar.gz
+  tar -xf "ga-${gaversion}.tar.gz"
 
-  cd ga-5.8
+  cd ga-${gaversion}
 
   # Build GA
-  echo "Building GA-5.8"
-  ./configure --with-mpi-ts --disable-f77 --without-blas --enable-cxx --enable-i4 --prefix=${PWD}/install_for_gridpack --enable-shared
+  echo "Building GA-${gaversion}"
+  ./configure --with-mpi-ts --disable-f77 \
+              --without-blas --without-lapack --without-scalapack \
+              --enable-cxx --enable-i4 \
+              --prefix=${PWD}/install_for_gridpack \
+              CFLAGS=-Wno-implicit-function-declaration \
+              $gashopts
   
   # Install GA
-  echo "Installing GA-5.8"
+  echo "Installing GA-${gaversion}"
   make -j 10 install
   
-  echo "GA-5.8 installation complete"
+  echo "GA-${gaversion} installation complete"
 fi
 
 if ${install_petsc}
 then
+
+  if [ "$install_shared" == "true" ]; then
+      petscopts="--with-shared-libraries=1"
+  else
+      petscopts="--with-shared-libraries=0"
+  fi
+
+  # petsc_version="3.21.4"
+  petsc_version="3.20.6"
     
-  # Install PETSc 3.16.4
+  # Install PETSc 
   cd ${GP_EXT_DEPS}
 
+  rm -rf petsc
+
   # Download
-  echo "Downloading PETSc 3.16.4"
```

Comment:
We need to move this upto 3.20. I believe for emt-alpha branch I am using 3.20.

Reply:
You show the original script code here.  The updated script uses 3.20.6, but 3.21.4 also worked for me.  It is much easier to change the version in the updated script.